Parsers/ASimDns/Parsers/ASimDnsSentinelOne.yaml

```diff
@@ -0,0 +1,196 @@
+Parser:
+  Title: DNS activity ASIM parser for SentinelOne
+  Version: '0.1.0'
+  LastUpdated: Jun 28 2023
+Product:
+  Name: SentinelOne
+Normalization:
+  Schema: Dns
+  Version: '0.1.7'
+References:
+- Title: ASIM DNS Schema
+  Link: https://aka.ms/ASimDnsDoc
+- Title: ASIM
+  Link: https://aka.ms/AboutASIM
+- Title: SentinelOne Documentation
+  Link: https://<SOneInstanceDomain>.sentinelone.net/api-doc/overview
+Description: |
+  This ASIM parser supports normalizing SentinelOne logs to the ASIM DNS normalized schema. SentinelOne events are captured through SentinelOne data connector which ingests SentinelOne server objects such as Threats, Agents, Applications, Activities, Policies, Groups, and more events into Microsoft Sentinel through the REST API.
+ParserName: ASimDnsSentinelOne
+EquivalentBuiltInParser: _ASim_Dns_SentinelOne
+ParserParams:
+  - Name: disabled
+    Type: bool
+    Default: false
+ParserQuery: |
+  let ThreatConfidenceLookup_undefined = datatable(
+      alertInfo_analystVerdict_s: string,
+      ThreatConfidence_undefined: int
+  )
+  [
+      "FALSE_POSITIVE", 5,
+      "Undefined", 15,
+      "SUSPICIOUS", 25,
+      "TRUE_POSITIVE", 33 
+  ];
+  let ThreatConfidenceLookup_suspicious = datatable(
+      alertInfo_analystVerdict_s: string,
+      ThreatConfidence_suspicious: int
+  )
+  [
+      "FALSE_POSITIVE", 40,
+      "Undefined", 50,
+      "SUSPICIOUS", 60,
+      "TRUE_POSITIVE", 67 
```

Comment:
any specific reason you kept this as 67 and not higher value incase of Suspicious TruePositive? e.g. 75

Reply:
As per our past experience we have defined the main category in three range as below:
- Undefined : 0~33
- Suspicious: 34~67
- Malicious: 68~100

And then assigned final confidence score based on the combination of "ruleInfo_treatAsThreat_s" and "alertInfo_analystVerdict_s" value as below:
<html>
<body>
<!--StartFragment--><google-sheets-html-origin><!--td {border: 1px solid #cccccc;}br {mso-data-placement:same-cell;}-->

ruleInfo_treatAsThreat_s | alertInfo_analystVerdict_s | Confidence
-- | -- | --
UNDEFINED | FALSE_POSITIVE | 5
||Undefined | 15
||SUSPICIOUS | 25
||TRUE_POSITIVE | 33
Suspicious | FALSE_POSITIVE | 40
||Undefined | 50
||SUSPICIOUS | 60
||TRUE_POSITIVE | 67
Malicious | FALSE_POSITIVE | 75
||Undefined | 80
||SUSPICIOUS | 90
||TRUE_POSITIVE | 100

<!--EndFragment-->
</body>
</html>